include/deal.II/lac/trilinos_solver.h

```diff
@@ -114,25 +114,19 @@ namespace TrilinosWrappers
        * base class for all solvers.
        */
       explicit
-      AdditionalData (const bool         output_solver_details   = false,
-                      const unsigned int gmres_restart_parameter = 30);
```

Comment:
Do you have a small test case that you're using to check the output of the solver? If so, could you share it and I'll take a look at this afternoon. I'm clearly not understanding why there's an issue, so it would be better if I just get my hands dirty and fiddle with it as well :-)